pkg/pillar/types/zedroutertypes.go

```diff
@@ -3169,6 +3169,24 @@ func GetIPAddrCountOnSubnet(subnet net.IPNet) int {
 	return 0
 }
 
+// GetIPAddrCountInRange returns count of addresses between two of them
+func GetIPAddrCountInRange(ip1, ip2 net.IP) int {
```

Comment:
Isn't there already a function which does this?

Reply:
Cannot find it, we have function for subnet, but not for IP range. I moved function to the correct place.